### ISSUE TITLE
LRN for InceptionV1?

### ISSUE THREAD

**Issue by swghosh** [tensorflow/hub#672]
I inspected the model being used by "https://tfhub.dev/google/imagenet/inception_v1/classification/1" (looked up the TF-slim implementation, since the SavedModel is ported from there), appears to me that the model is using Batch Normalization layer instead of LRN layer discussed in the original paper (InceptionV1 a.k.a. GoogLeNet)

**Comment by arnoegw** [comment]
That's correct.

**Comment by swghosh** [comment]
So like it's still an official implementation, right? Not much harm w/ using BatchNormalization instead of LRN.
I think so that w/ BatchNorm results had improved also.

**Comment by arnoegw** [comment]
I'm not sure what "official" would mean here... In any case, TF-Slim is a reputable source of CNNs and has been widely used. It provides a credible implementation of the Inception V1 architecture, albeit extended by batch normalization, as you say, which does improve training.

Is there a specific issue with TF Hub that you would like the TF Hub team to address?

**Comment by swghosh** [comment]
I think that's a reasonably good explanation. I was unsure about creating issues on the tf-slim repository as it not maintained anymore. Thanks for the support anyway!

**Comment by arnoegw** [comment]
Happy to help.

BTW, version /1 is outdated. I recommend using https://tfhub.dev/google/imagenet/inception_v1/classification/3 with TF1/`hub.Module` or https://tfhub.dev/google/imagenet/inception_v1/classification/4 with TF1 and the new APIs for [TF2 SavedModels](https://www.tensorflow.org/hub/tf2_saved_model).

Our previous discussion applies equally.

**Comment by swghosh** [comment]
Yeah, am aware of that. I was using "/4", just moght have shared the wrong link.
